### ISSUE TITLE
Latest data is not shown after CoE Upgrade

### ISSUE THREAD

**Issue by svc-o365-group-prod** [microsoft/coe-starter-kit#8950]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Hi,

We did a CoE upgrade on 23rd August and upgraded of all components were successful.
After the Upgrade, latest data is not shown either in Power Platform Admin view App or on Power BI CoE dashboards. 
I have run the Setup wizard after the upgrade also, but still the data is old.

### Expected Behavior

Power Platform Adminview App and Power BI Dashboards should show latest flow or App details.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.37

### What app or flow are you having the issue with?

Power Platform Admin View

### What method are you using to get inventory and telemetry?

Cloud flows

### Steps To Reproduce

Upgrade all components from latest August release, Once upgrade is successful.
Open Power Platform Admin view App and see if flows or Apps that are modified recently are shown. 

![CoESolution](https://github.com/user-attachments/assets/03a362a6-b22b-4d09-a3c8-75f62a7cda5c)

**Comment by Jenefer-Monroe** [comment]
1. Can you please clarify what you mean by new data being missing? Have you added an app/flow and dont see it next day?

2. Please go to the Admin Command Center > CoE Flows > Inventory. Validate that all flows are turned on and that there are no unmanaged layers as shown here.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/598b5024-2e1a-4390-a6b2-49644368c246)

3. See if there are any sync flow errors to investigate. 
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/d43b50b3-84d1-4894-a9a3-04d5d95fe8dc)

**Comment by svc-o365-group-prod** [comment]
Hi,

Just a quick update, we upgraded August release on top of March release.


  1.
When I search for flows and Apps that are updated recently, I see only data on and before 22-Aug.
We have lot of apps and flows that gets created or modified daily.
  2.
Some flows didn't turned on, so I have turned on below flows:
Below are the 2 flows that are turned off, I couldn't turn them On:
Admin | Sync Template v4 (Driver) -- when I try to turn on, I get below error message
Flow client error returned with status code "BadRequest" and details "{"error":{"code":"ChildFlowNeverPublished","message":"The workflow with id '7f01d8ae-ca50-ef11-a316-6045bd039c1f' cannot be used as a child workflow because it has never been published. Child workflows need to be published at least once before they can be included in a published parent workflow."}}".
Admin | Gather Tenant SRs –  This flow is On, but it fails with below error:
Action 'Get_Parent_BU_for_ID' failed: RetrievePrivilegeForUser: The user with id f1558604-f069-ef11-a670-0022489ebf33 has not been assigned any roles. They need a role with the prvReadBusinessUnit privilege


  3.
Few errors on Sync Audit Logs (V2) flow with below error:
Action 'Get_Azure_Secret' failed: Error occured while reading secret: Value cannot be null. Parameter name: input
[cid:4e5f9ef4-0987-4522-a855-17adab83e021]

I don't know if this could be an issue, we observed that Microsoft 365 Management API (for Audit logs) client secret got expired. we created new client secret and added to environment variables and started the 'Admin | Audit Logs | Office 365 Management API Subscription' flow.
https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-auditlog-http
[https://learn.microsoft.com/en-us/media/open-graph-image.png]<https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-auditlog-http>
Collect audit logs using an HTTP action - Power Platform<https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-auditlog-http>
Connect audit log sync flows to the audit log using an HTTP action in a cloud flow to gather telemetry data (unique users, launches) for apps in Microsoft 365.
learn.microsoft.com



Thanks,

Raj



________________________________

**Comment by Jenefer-Monroe** [comment]
Ok it sounds like you are not collecting inventory today as your driver is turned off. 
For issue 3, plesae post separately as that will not be related to your missing inventory. 

For your missing inventory, first please go validate that you dont have any unmanaged layers on the flows as shown above and remove any you see. 

Then boot the setup wizard and walk through its steps. It will ensure all the flows are turned on including the new child blocking your driver. 

Note that you should always use the setup wizard after an upgrade. 
Here is some more information about the setup wizard.

### Use the setup wizard
Its highly recommended that you use the Setup Wizard to walk you through setup and upgrades. It will ensure all the new flows are turned on in the correct order, trigger things like population of the bookmarks screen and do other cleanup steps needed.
You should do this both for upgrades and clean installs.
To use it, first perform the import of the solution install or upgrade manually, then boot the app and walk through the screens.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/b84cc1ff-383a-497c-9236-569e8313c73c)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/ed02f0d0-c276-48cc-90af-47e5bf5a928e)

**Comment by svc-o365-group-prod** [comment]
Thanks Jenefer,
Now all data flows in and I could see updated data in both CoE Dashboards and Power Platform Admin View App.
After fixing the issues, reran setup wizard and everything works smoothly now.

No Sync Flow Errors and In CoE Flows –> Inventory - all flows are Turned On (Only Add Maker to Group is turned off intentionally).

Only thing that annoys is when I open CoE Admin Command Center, it shows this warning message, even though I upgraded to August release.
[cid:81d10a65-2426-4d04-a246-3a952a33e807]

[cid:d81569cb-ec67-4a50-8f07-7dfa39c41c1c]


Thanks,

Raj



________________________________

**Comment by Jenefer-Monroe** [comment]
You picture did not come through so I'm not sure but there is a Sept release now so you will get notified of that in these tools. 
Glad you got unblocked!

**Comment by svc-o365-group-prod** [comment]
Now attached images, 
I get message like CoE Starter kit solution was installed on Mar 2024. Install latest version..

![image](https://github.com/user-attachments/assets/97556d36-a2e1-4966-89fe-9fb340146f31)

My Solution versions are already on August version.
![image](https://github.com/user-attachments/assets/7bbe919d-4018-4a32-bf14-55ff37b9e479)

**Comment by Jenefer-Monroe** [comment]
yep looks like your created date isnt updating for some reason. 
You can feel free to post that as a new issue if you'd like but unfortunately it will sit on the backlog a while as we are heads down in the Data Export V2 world for the next ~6 months.
